frontend/webapp/core/__init__.py

```diff
@@ -47,7 +49,8 @@
 csp = {
     "default-src": "'self'",
     "script-src": "'self'",
-    "connect-src": ["'self'", "stat.ripe.net"],
+    # connect-src self doesn't work for wss. Ticket at https://bugs.webkit.org/show_bug.cgi?id=201591
+    "connect-src": ["'self'", "stat.ripe.net", "wss://{}".format(ARTEMIS_WEB_HOST)],
```

Comment:
Does this create any issues for the non-safari browsers? (chrome, FF)?

Reply:
nope